### PR TITLE
Some spots on lavaland no longer produce asteroid flooring

### LINKED ISSUE #16849: The high chance ore spots scattered about lavaland are the wrong type of mineral turf
They break to normal asteroid turf (and probably space if you bomb them).

@ChangelingRain

### PR DESCRIPTION
**Pull request by ChangelingRain**
Fixes #16849
Also fixes low GPS amount.

### CODE REVIEW

File: _maps/map_files/generic/lavaland.dmm [hunk 1]
```diff
@@ -14130,7 +14132,7 @@ af
 af
 ab
 ee
-en
+em
```
Comment (xxalpha) [inline _maps/map_files/generic/lavaland.dmm]: Were you swapping tiles around?

Comment (xxalpha) [inline _maps/map_files/generic/lavaland.dmm]: Also did you use mapmerge script or was this a manual edit?

**Comment by KorPhaeron** [comment]
Thanks

**Comment by ChangelingRain** [comment]
Mapmerge, I don't know how to do manual.

**Pull request merged**